src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -469,7 +469,7 @@ public partial class JsonConverterAttribute : System.Text.Json.Serialization.Jso
     public abstract partial class JsonConverterFactory : System.Text.Json.Serialization.JsonConverter
     {
         protected internal JsonConverterFactory() { }
```

Comment:
Not related to your change, but why is the ctor protected internal? What benefit is the internal part providing?

Reply:
@steveharter?

Reply:
It isn't needed - on an abstract class an "internal" ctor will never be called AFAIK.

Reply:
Exactly.